components/layouts/TheMenu/index.vue

```diff
@@ -8,41 +8,38 @@
       <div class="menu__overlay" />
       <SidebarMenu class="menu__side-menu" />
     </section>
-    <div class="menu__main-level-wrapper">
-      <section class="menu__main-level">
-        <nav class="menu__nav-section">
-          <AppLink class="menu__link" v-bind="homeLink">
-            {{ homeLink.label }}
-          </AppLink>
-          <div class="menu__separator" />
-          <AppLink
-            v-for="link in qiskitElements"
-            :key="link.url"
-            class="menu__link"
-            :class="{ 'menu__link_active': isActive(link) }"
-            v-bind="link"
-          >
-            {{ link.label }}
-          </AppLink>
-        </nav>
-        <nav class="menu__nav-section">
-          <AppLink
-            v-for="link in [communityLink, ...learnMore]"
-            :key="link.url"
-            class="menu__link"
-            :class="{ 'menu__link_active': isActive(link) }"
-            v-bind="link"
+    <section class="menu__main-level">
+      <nav class="menu__navigation-level">
+        <AppLink
+          class="
+            menu__link
+            menu__link_img-only
```

Comment:
I'm trying to find the effect of this class `menu__link_img-only` and I didn't find it. I'm missing something or this class is unnecessary?

Reply:
In theory, it removed some extra space added by the line height of the link but that is completely nullified by the fact that links' display is flex in column mode. Should be removed.